deploy/helm/sumologic/values.yaml

```diff
@@ -106,126 +71,279 @@ sumologic:
     cacheRefresh: "1800"
 
+  logs:
+    ## Configuration for sumologic output plugin
+    ## ref: https://github.com/SumoLogic/fluentd-output-sumologic
+    ## ref: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/master/deploy/helm/sumologic/conf/logs/logs.output.conf
+    output:
+      ## Format to post logs into Sumo: fields, json, json_merge, or text.
+      ## NOTE: for logs metadata, fields is required.
+      logFormat: fields
+      ## Option to control adding timestamp to logs.
+      addTimestamp: true
+      ## Field name when add_timestamp is on.
+      timestampKey: "timestamp"
+      ## Additional config parameters for sumologic output plugin
+      extraConf: |-
+
+    ## Additional config for custom log pipelines
+    ## ref: TODO: documentation for custom logs pipelines
+    extraLogs: |-
+
+    ## Container log configuration
+    containers:
+      outputConf: |-
+        @include logs.output.conf
+      ## Set the _sourceName metadata field in Sumo Logic.
+      sourceName: "%{namespace}.%{pod}.%{container}"
+      ## Set the _sourceHost metadata field in Sumo Logic.
+      sourceHost: ""
+      ## Set the _sourceCategory metadata field in Sumo Logic.
+      sourceCategory: "%{namespace}/%{pod_name}"
+      ## Set the prefix, for _sourceCategory metadata.
+      sourceCategoryPrefix: "kubernetes/"
+      ## Used to replace - with another character.
+      sourceCategoryReplaceDash: "/"
+
+      ## A regular expression for containers.
+      ## Matching containers will be excluded from Sumo. The logs will still be sent to FluentD.
+      excludeContainerRegex: ""
+      ## A regular expression for hosts.
+      ## Matching hosts will be excluded from Sumo. The logs will still be sent to FluentD.
+      excludeHostRegex: ""
+      ## A regular expression for namespaces.
+      ## Matching namespaces will be excluded from Sumo. The logs will still be sent to FluentD.
+      excludeNamespaceRegex: ""
+      ## A regular expression for pods.
+      ## Matching pods will be excluded from Sumo. The logs will still be sent to FluentD.
+      excludePodRegex: ""
+
+      ## ref: https://github.com/fabric8io/fluent-plugin-kubernetes_metadata_filter#configuration
+      k8sMetadataFilter:
+        ## Option to control the enabling of metadata filter plugin watch.
+        watch: "true"
+        ## path to CA file for Kubernetes server certificate validation
+        caFile: ""
+        ## Validate SSL certificates
+        verifySsl: true
+        ## Path to a client cert file to authenticate to the API server
+        clientCert: ""
+        ## Path to a client key file to authenticate to the API server
+        clientKey: ""
+        ## Path to a file containing the bearer token to use for authentication
+        bearerTokenFile: ""
+
+      ## To use additional filter plugins
+      extraFilterPluginConf: |-
+
+    ## Kubelet log configuration
+    kubelet:
+      outputConf: |-
```

Comment:
@samjsong do you want to add the `enabled` flags for the log types in this PR itself?

Reply:
let's include these in a separate PR if we choose to do this 👍